lib/api/BackbeatAPI.js

```diff
@@ -79,6 +80,9 @@ class BackbeatAPI {
         if (route.substring(3) === 'healthcheck') {
             return true;
         }
+        if (route.substring(3) === 'monitoring/metrics') {
```

Comment:
On one hand, I understand that we want to separate prometheus metrics from raw CRR metrics, since prometheus has its own logic of handling it, on the other hand, I can find confusing for the user to see the `/_/metrics` and `/_/monitoring/metrics` routes. Is the end goal to expose everything through prometheus and get rid of other `/_/metrics/xxx` routes? Or are they meant to co-exist?

Maybe a `/_/metrics/monitoring` route could make sense then? Not sure, just raising the question, it's completely open to discussion.

It may not be very important but when the product is getting complex with more and more routes it can get messy quickly if we don't plan for some good organization, and it becomes harder to change once applications start using the routes.

Reply:
While discussing with Anurag, I thought it is definitely possible to have prometheus track all metrics since it does the same operations of incrementing the 4 data points we collect. I'm not sure of performance, but a great thing about the promclient was throughput can be represented more accurately rather than the current representation through an average of a period of time (15 minutes by default).

Didn't look too far into promclient but supporting site level can be potentially hard to implement. Not sure if we can specify a site name to each increment.

For now, the promclient will have to sit separate, so maybe to address your concerns on routes, we can change the route name for the prometheus metrics.